src/Mesh/babylon.groundMesh.ts

```diff
@@ -5,6 +5,8 @@
         private _worldInverse = new Matrix();
         private _heightQuads: { slope: Vector2; facet1: Vector4; facet2: Vector4 }[];
         public _subdivisions: number;

```

Comment:
isn't subdivisions redundant? It is actually not needed anymore (here, I am not talking about the meshBuilder).